faq.html

```diff
@@ -164,141 +172,143 @@ <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="How_much_does_it_cos
-      transfer is considered in breach of contract, which will result in dispute resolution.</p>
-    <p>To avoid problems we recommend setting up a dedicated bank account for buying and selling bitcoins. If the bank were to block this account, it would cause fewer problems for you than if the bank were to block your primary account.</p>
-    <p>Never mention anything else than the trade reference in the subject field.</p>
+    <p>There have been reports that some banks have closed client accounts when they sense Bitcoin activity.</p>
+    <p>The reference text (sometimes called "reason for payment" or similar) of the bank transfer used to trade with Bisq must always <span class='font-italic'>only</span> include the Bisq trade ID, which is a random sequence of characters, just like many other purchase IDs. Putting anything else as a "reason for payment" is considered a violation of the trade protocol and will result in a dispute.</p>
+    <p>To avoid problems, we recommend setting up a dedicated bank account for buying and selling bitcoins. If the bank were to block this account, it would cause fewer problems for you than if the bank were to block your primary account.</p>
+    <p>Never mention anything else other than the Bisq trade ID in the subject field.</p>
   </div>
 </div>
 
 
 <h4 class="pt-5 mt-5 pb-3">Dispute Resolution</h4>
 <div class="accordion border-bottom mb-5">
 
-  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="What_is_an_arbitrator">What is an arbitrator?</h6>
+  <h6 class="accordion-toggle border-top pt-3 pb-2 green" id="arbitrators">What is an arbitrator?</h6>
   <div class="accordion-content pb-5 col-sm-12 col-md-8 ml-0 pl-sm-0 pl-md-0 px-0">
-    <p>An arbitrator is the person who find the solution to a dispute that may occur during a trade. <a href="/docs/exchange/arbitration-system" target="_blank">Bisq&#8217;s arbitrators</a> must pay a <a href="https://docs.bisq.network/dao/phase-zero.html" target="_blank" rel="noopener">BSQ bond</a>, which ensure fair evaluations and prevent collusion with traders.</p>
-    <p>Initially, traders make a list of arbitrators they approve. When a new trade is initiated, an arbitrator is randomly selected from among the traders&#8217; overlapping, approved selections. The more arbitrators a trader accepts, the more trades will be available to him.</p>
+    <p>Arbitrators determine solutions to trade disputes. Bisq&#8217;s arbitrators must pay a very high <a href="https://docs.bisq.network/dao/phase-zero.html" target="_blank" rel="noopener">BSQ bond</a>, which ensures fair evaluations and prevents collusion with traders.</p>
+    <p>Arbitrators are chosen randomly for each trade.</p>
+    <p>Soon after the DAO launch, the arbitrator system will be phased out with the launch of <a href='https://github.com/bisq-network/proposals/issues/52' target='_blank'>a new trade protocol</a> that replaces arbitrators with mediators for better dispute resolution.</p>
```

Comment:
We had recently changed in the new trade protocol to keep the arbitration for a while as optional feature. 

Maybe just don't mention it here as the proposal is not updated as well.